src/utils/const.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable import/prefer-default-export */
+const CHAPTERS_DETAILS = [
+  { versesCount: 7 },
+  { versesCount: 286 },
+  { versesCount: 200 },
+  { versesCount: 176 },
```

Comment:
I do like the local cache approach, though we should do the following:

1) Create a separate directory outside of src and call it `data`
2) Create a file called under data `Chapters.json`, this file will contain all 114 chapters as a json blob. with the key being the chapter number, and one the children being the verses count. 

In the future, we can add more data to be something like this: 

<img width="685" alt="Screen Shot 2021-07-19 at 11 23 20 AM" src="https://user-images.githubusercontent.com/11590314/126136965-0bfc793a-c9b3-40cc-876c-15fb975cceeb.png">